### PR TITLE
lofis for emergency contact flow and revisions to connect device screen

### PR DESCRIPTION
**Pull request by caterinawang**
Lo-fis for emergency contact screen and add contact screen and revisions to connect device screen - does not make backend calls yet

### CODE REVIEW

**Comment by denk0403** [comment]
I think merging in your previous PR created merge conflicts with this one. Would you be able to fix them?

File: src/screens/AddContactScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+export default function AddContactScreen() {
+
+    return (
+        <View style={styles.container}>
+            <View style={{ flex: 1, alignItems: 'center' }}>
+                <View style={{ flexDirection: 'row', marginTop: 50, marginBottom: 25, justifyContent: 'center' }}>
+                    <View style={{ marginLeft: '7%' }}>
+                        <TouchableOpacity>
+                            <FontAwesome name="long-arrow-left" size={28} color="black" />
```
[the remark this answers is not in the document]
Comment (caterinawang) [reply src/screens/AddContactScreen.tsx:15]: agreed - gonna implement built in navigation after our talk @ sprint review and connect to backend before I merge I'll take a look at merge conflicts

**Review by mattognibene — COMMENTED** [review]
Testing this on my android emulator, clicking the connect device button doesn't seem to navigate anywhere. Any ideas what might be causing this?

**Comment by denk0403** [comment]
> Testing this on my android emulator, clicking the connect device button doesn't seem to navigate anywhere. Any ideas what might be causing this?

I think the button isn't wired-up with navigation yet.

**Comment by caterinawang** [comment]
> Testing this on my android emulator, clicking the connect device button doesn't seem to navigate anywhere. Any ideas what might be causing this?

@mattognibene still struggling w navigation - currently working through getting that done

**Review by denk0403 — APPROVED** [review]

**Pull request merged**